src/bp/nlu/typings_v1.d.ts

```diff
@@ -1,21 +1,31 @@
 // Typings for Stan's API v1
 
+export interface Specifications {
+  nluVersion: string // semver string
+  languageServer: {
+    dimensions: number
+    domain: string
+    version: string // semver string
+  }
+}
+
+export interface Health {
+  isEnabled: boolean
+  validProvidersCount: number
+  validLanguages: string[]
+}
+
 /**
- * ################################
- * ############ INPUTS ############
- * ################################
+ * ##################################
+ * ############ TRAINING ############
+ * ##################################
  */
-export interface TrainInput extends Credentials {
+
+export interface TrainInput {
   language: string
-  contexts: string[]
   intents: IntentDefinition[]
-  entities: (ListEntityDefinition | PatternEntityDefinition)[]
-  seed?: number
-}
-
-export interface Credentials {
-  appId: string
-  appSecret: string
+  entities: EntityDefinition[]
+  seed: number
```

Comment:
Seed is optional in `TrainRequestBody` in `http-typings` but required here, do we choose a default value somewhere?

Reply:
yes, in the `api.ts`. The variable is called `pickedSeed`

Reply:
l-166: `const pickedSeed = seed ?? Math.round(Math.random() * 10000)`